content/docs/studio/troubleshooting.md

```diff
@@ -188,6 +250,22 @@ not in the hidden commits list, please [raise a support request](#support).
 [display preferences -> hide commits]:
   /doc/studio/user-guide/projects-and-experiments/explore-ml-experiments#hide-commits
 
+## Error: Missing metric or plot file(s)
+
+This error message means that the metric or plot files referenced from
+`dvc.yaml` could not be found in your Git repository or cache. Make sure that
+you have pushed the required files using `dvc push`. Then try to import the
+repository again.
+
+## Error: Skipped big remote file(s)
+
+Files that are larger than 512 KB are currently skipped by Iterative Studio.
```

Comment:
@amritghimire Can you pls update the current value of this limit.

Reply:
It is 10MB